test/distributed/elastic/rendezvous/c10d_rendezvous_backend_test.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from datetime import timedelta
+from typing import ClassVar
+from unittest import TestCase
+
+from torch.distributed import TCPStore
```

Comment:
A bit late to this PR, but is the goal of `C10dRendezvousBackend` to be able to support other stores such as FileStore and HashStore as well, or only TCPStore?

Reply:
As of today it only supports `TCPStore` (read it as "only tested with"), but it is implemented in a generic way and can potentially be used with any `Store` type that has a `compare_set` function. In fact Howard recently implemented `compare_set` for `FileStore`. This means technically we can also use it with `FileStore`.